etcdserver/etcdserverpb/rpc.proto

```diff
@@ -385,6 +385,67 @@ service Auth {
   }
 }
 
+
```

Comment:
Please document the methods in comments in detail. 
In particular mark them as available since etcd 3.5+

Reply:
thanks, I will do it. The purpose of POC is to confirm whether the community agrees to add this feature and how to implement it. If the community agrees with this feature and roughly agrees with the implementation of this POC solution, I will continue to improve it and open a detailed task list issue , split it into small PRs, add documentation, comments, tests, etc. anyone interested in this feature can join in the development. thanks for your review.